llvm/lib/Bitcode/Reader/MetadataLoader.cpp

```diff
@@ -60,9 +60,6 @@
 #include <type_traits>
 #include <utility>
 #include <vector>
-namespace llvm {
```

Comment:
This one is unrelated to the other changes and can be split into a separate PR and committed. There is a use of Argument in this file, but presumably it is being pulled in via other headers, so no need for this forward def.